apps/schools/domains/common/components/backPage/index.tsx

```diff
@@ -1,13 +1,18 @@
 import React from 'react'
-import { Row, Typography } from 'antd'
+import { Typography } from 'antd'
 import styles from './styles/styles.module.scss'
 import { BackPageProps } from './interface'
 import classNames from 'classnames'
 import { useRouter } from 'next/router'
 import { ArrowLeftOutlined } from '@ant-design/icons'
 
-export const BackPage: React.FC<BackPageProps> = ({ className, path, onClick, text = 'Назад' }) => {
+export const BackPage: React.FC<BackPageProps> = ({ className, path, onClick, text = 'Назад', returnable = false }) => {
     const router = useRouter()
+    const { back } = router.query
```

Comment:
doubtful, but okay

Reply:
you could use use special lib for that

Reply:
another library instead of the main functions?